flocker/node/agents/blockdevice.py

```diff
@@ -66,6 +66,10 @@ class DatasetStates(Names):
     """
     # Doesn't exist yet.
     NON_EXISTENT = NamedConstant()
+    # Existing volume not recorded as owning a dataset
+    UNREGISTGERED = NamedConstant()
+    # Ownership recorded but not reported as a volume
+    REGISTERED = NamedConstant()
```

Comment:
In particular, might be worth mentioning when this can happen (eventual consistency in AWS where EBS volume was created but isn't being listed reliably, I assume...)